browser/plugins/three_vr_camera.plugin.js

```diff
@@ -46,6 +46,8 @@
 
 		this.rotationFromGraph = new THREE.Euler()
 		this.positionFromGraph = new THREE.Vector3()
+
+		this.tempRotationEuler = new THREE.Euler()
```

Comment:
this isn't so "temp" actually, but maybe more like a perspectiveCameraRotationEuler? or a proxy even?